### PR TITLE
Stuffs

### PR DESCRIPTION
**Pull request by NycuRO**

### CODE REVIEW

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java [hunk 1]
```diff
@@ -140,7 +140,7 @@ public boolean onUpdate(int currentTick) {
                         this.setDataFlag(DATA_FLAGS, DATA_FLAG_IGNITED, true);
                     }
                     this.bombTime += tickDiff;
-                    if (this.bombTime >= 64) {
+                    if (this.bombTime >= 34) {
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java]: I think that is bit too fast

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java [hunk 2]
```diff
@@ -95,7 +95,7 @@ public boolean onUpdate(int currentTick) {
         }
 
         if (!this.isAlive()) {
-            if (++this.deadTicks >= 23) {
+            if (++this.deadTicks >= 23  ) {
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java]: Why space?

File: src/main/java/nukkitcoders/mobplugin/MobPlugin.java [hunk 3]
```diff
@@ -170,7 +173,7 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
     }
 
     private void registerEntities() {
-        BlockEntity.registerBlockEntity("MobSpawner", BlockEntitySpawner.class);
+        //BlockEntity.registerBlockEntity("MobSpawner", BlockEntitySpawner.class);
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/MobPlugin.java]: Why?

File: src/main/java/nukkitcoders/mobplugin/MobPlugin.java [hunk 4]
```diff
@@ -83,7 +85,8 @@ public void onEnable() {
         int spawnDelay = pluginConfig.getInt("entities.autospawn-ticks", 0);
 
         if (spawnDelay > 0) {
-            this.getServer().getScheduler().scheduleDelayedRepeatingTask(this, new AutoSpawnTask(this), spawnDelay, spawnDelay);
+            this.getServer().getScheduler().scheduleRepeatingTask(this, new NetherWorldMobsSpawnTask(), 20 * 5, true);
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/MobPlugin.java]: Shouldn't this use spawn delay from config?

File: src/main/java/nukkitcoders/mobplugin/entities/BaseEntity.java [hunk 5]
```diff
@@ -8,6 +8,7 @@
 import cn.nukkit.event.entity.EntityDamageByEntityEvent;
 import cn.nukkit.event.entity.EntityDamageEvent;
 import cn.nukkit.item.Item;
+import cn.nukkit.level.Position;
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/entities/BaseEntity.java]: For what?

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java (hunk 1 above)
Comment (NycuRO) [reply src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java]: bomb time is animation when creeper start blowing, not when start be flinted, idk how explain

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java (hunk 2 above)
Comment (NycuRO) [reply src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java]: intel

File: src/main/java/nukkitcoders/mobplugin/MobPlugin.java (hunk 3 above)
Comment (NycuRO) [reply src/main/java/nukkitcoders/mobplugin/MobPlugin.java]: i forgot why, i will edit

File: src/main/java/nukkitcoders/mobplugin/MobPlugin.java (hunk 4 above)
Comment (NycuRO) [reply src/main/java/nukkitcoders/mobplugin/MobPlugin.java]: not really it's ok

File: src/main/java/nukkitcoders/mobplugin/entities/BaseEntity.java (hunk 5 above)
Comment (NycuRO) [reply src/main/java/nukkitcoders/mobplugin/entities/BaseEntity.java]: intel

**Comment by PetteriM1** [comment]
Btw is the the jar file size supposed to be over 14 mb now?

File: src/main/java/nukkitcoders/mobplugin/spwning/NetherWorldMobsSpawnTask.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+    private static ThreadLocal<HashSet<FullChunk>> spawnChunks = ThreadLocal.withInitial(HashSet::new);
+
+    @SuppressWarnings("unchecked")
+    private static Class<? extends Entity>[]
+        monsters_nether = new Class[] {
+            Blaze.class,
+            Ghast.class,
+            ZombiePigman.class,
+            ZombiePigman.class,
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/spwning/NetherWorldMobsSpawnTask.java]: How about wither skeletons? Also isn't endermans spawned on nether too?

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java (hunk 1 above)
Comment (PetteriM1) [reply src/main/java/nukkitcoders/mobplugin/entities/monster/walking/Creeper.java]: Actually checked this from wiki and it's supposed to be 1.5 seconds (30 ticks)

**Comment by PetteriM1** [comment]
So you are not going to continue this?

**Comment by NycuRO** [comment]
nop

**Pull request closed without merging**